scipy/stats/_multivariate.py

```diff
@@ -4563,12 +4564,32 @@ def _entropy(self, dim, df=1, shape=1):
             return multivariate_normal(None, cov=shape).entropy()
 
         shape_info = _PSD(shape)
-        halfsum = 0.5 * (dim + df)
-        half_df = 0.5 * df
-        return (-gammaln(halfsum) + gammaln(half_df)
+        shape_term = 0.5 * shape_info.log_pdet
+
+        def regular(dim, df):
+            halfsum = 0.5 * (dim + df)
+            half_df = 0.5 * df
+            return (
+                -gammaln(halfsum) + gammaln(half_df)
                 + 0.5 * dim * np.log(df * np.pi) + halfsum
                 * (psi(halfsum) - psi(half_df))
-                + 0.5 * shape_info.log_pdet)
+                + shape_term
+            )
+
+        def asymptotic(dim, df):
+            # Formula from Wolfram Alpha:
+            # "asymptotic expansion -gammaln((m+d)/2) + gammaln(d/2) + (m*log(d*pi))/2
+            #  + ((m+d)/2) * (digamma((m+d)/2) - digamma(d/2))"
+            return (
+                dim * norm._entropy() + dim / df
+                - dim * (dim - 2) * df**-2.0 / 4
+                + dim**2 * (dim - 2) * df**-3.0 / 6
+                + dim * (-3 * dim**3 + 8 * dim**2 - 8) * df**-4.0 / 24
+                + shape_term
+            )
```

Comment:
This is currently returning a 0d array. Ensure that it returns a scalar:
```suggestion
            )[()]
```